pkg/resources/statefulsets/statefulset.go

```
@@ -64,6 +83,32 @@ func createElasticsearchMasterStatefulSet(log vzlog.VerrazzanoLogger, vmo *vmcon
 		// HTTP is enabled on the master here solely for our readiness check below (on _cluster/health)
 		{Name: "HTTP_ENABLE", Value: "true"},
 		{Name: "logger.org.opensearch", Value: "info"},
+		{Name: constants.OciAccessKeyVarName,
```

Would it be better if these were mounted as optional volumes?

---

So the script does a check whether this value is availble. With this being enforced atleast we will see a pod configuration error. Otherwise things mayfail silently and even the script wont complain